src/services/user.js

```diff
@@ -96,6 +99,10 @@ const userService = {
       throw createError(409, ALREADY_REGISTERED)
     }
 
+    if (role === ADMIN) {
+      throw createBadRequestError()
+    }
+

```

Comment:
I think in this case (if a regular user attempts to log in with an admin role), an appropriate error would be a 403 Forbidden error. This indicates that the server understood the request but refuses to authorize it because the user is trying to assume a role they are not permitted to have

Reply:
Yes, but in this case we are creating new user, not logging in the existing one